packages/cli-template-monorepo-ethers/apps/web-app/.env.production

```diff
@@ -1,5 +1,8 @@
 NEXT_PUBLIC_DEFAULT_NETWORK=sepolia
-NEXT_PUBLIC_FEEDBACK_CONTRACT_ADDRESS=
+NEXT_PUBLIC_INFURA_API_KEY=abf67af1010b4b8d877e04244f1eac3d
```

Comment:
Since this is a template, I think it's ok to keep `NEXT_PUBLIC_FEEDBACK_CONTRACT_ADDRESS` and `NEXT_PUBLIC_GROUP_ID` with empty values in the `.env.production` file. (for both templates: ethers and subgraph)

Reply:
Modified in both templates